### DISCUSSION TITLE
query in queue....

### DISCUSSION THREAD

**Discussion by yehengda** [getredash/redash#6668] [Q&A] [answered]
I am trying to connect to my MySQL database. The test connection shows success, and it also displays my data tables. However, when I execute a simple query like SELECT 1, it always shows "query in queue...". I have checked my logs as shown in the attached image (Image 1). Could anyone help me solve this issue? I installed Redash using Docker, and my MySQL database is running in a Windows 10 environment.
![log](https://github.com/getredash/redash/assets/34308100/76f0430a-91d6-4033-9612-f8cce19242ce)

![redash](https://github.com/getredash/redash/assets/34308100/91b98561-e58e-43d5-8c02-d690c08c573f)

yml
version: '2'
services:
  server:
    image: redash/redash:latest
    command: server
    depends_on:
      - postgres
      - redis
    ports:
      - "5000:5000"
    environment:
      REDASH_REDIS_URL: "redis://redis:6379/0"
      REDASH_DATABASE_URL: "postgresql://postgres:password@postgres/postgres"
      REDASH_COOKIE_SECRET: veryverysecret
      REDASH_SECRET_KEY: veryverysecret
      REDASH_WEB_WORKERS: 100

  worker:
    image: redash/redash:latest
    command: scheduler
    depends_on:
      - server
    environment:
      REDASH_REDIS_URL: "redis://redis:6379/0"
      REDASH_DATABASE_URL: "postgresql://postgres:password@postgres/postgres"
  redis:
    image: redis:5.0-alpine
    restart: always
  postgres:
    image: postgres:9.6-alpine
    volumes:
      - ./data/postgres:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: password

**Comment by masayuki038** [comment, marked as answer]
@yehengda `redash/redash:latest` is a container created 4 years ago. I don't know the Redash version. However, it may be v8.
![image](https://github.com/getredash/redash/assets/36331/8b745dd7-85db-41ab-92c8-4d1fda35b394)

I believe that this docker-compose.yml is for v8. It looks like there are some parameters for "QUEUES". Could you try this?
https://github.com/getredash/setup/blob/master/data/docker-compose.yml

**Comment by hanzhenling** [comment]
the github link is gone. How TO SOVLE IT THEN ? THANKS
